src/Compilers/Core/Portable/InternalUtilities/CompilerOptionParseUtilities.cs

```diff
@@ -20,13 +20,7 @@ public static IList<string> ParseFeatureFromMSBuild(string features)
             }
 
             var all = features.Split(new[] { ';', ',', ' ' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
The variable had an obvious purpose in the previous iteration before your change. If it has no purpose now, it should probably be removed (inlined).

---
In reply to: [80747682](https://github.com/dotnet/roslyn/pull/13975#discussion_r80747682) [](ancestors = 80747682)